### PR TITLE
eslint: fix semicolon and some duplicate key

### PR DESCRIPTION
**Pull request by zalog**
- some ESLint semicolon validations
- some duplicate object key

Test plan: `grunt test`

### CODE REVIEW

**Review by Johann-S — COMMENTED** [review]
I don't see any interest to add semicolon to our ``Gruntfile.js`` because currently we do not use semicolon on our client side JS and in the others JS files

**Comment by zalog** [comment]
There are IDE like Visual Studio Code that comes with ESLint by default and every time we open `bootstrap` folder/project we get this validation errors that mess up with other possible real problems :)
If we decide to ignore this problems, they should be added in `.eslintignore` file.

File: js/.eslintrc.json [hunk 1]
```diff
@@ -72,7 +72,6 @@
     "no-new": "error",
     "no-new-func": "off",
     "no-new-wrappers": "error",
-    "no-new": "error",
```
Comment (Johann-S) [inline js/.eslintrc.json]: Good catch

**Comment by Johann-S** [comment]
Update ``.eslintignore`` should be a good way to do this.

@bardiharborow any concern about that ?

**Comment by zalog** [comment]
Sorry, later edit to be more clear.

On our generated client side JS files, like `js/dist/`, there are semicolons. I'm not sure, but probably babel do that.
But semicolons are not in `js/src`, ESLint does not require that because of some specific settings in `js/.eslintrc.json`.

**Comment by zalog** [comment]
Guys, do you want me to make another PR with the `no-new` duplicate key?
And rebase this PR without that and leave it here for further discussions?

**Comment by bardiharborow** [comment]
PR for this coming soon.

**Comment by bardiharborow** [comment]
Closing for #22477.

**Comment by zalog** [comment]
Yes, but still https://github.com/twbs/bootstrap/pull/22388#issuecomment-292885868

**Comment by bardiharborow** [comment]
Yep, I think maybe just moving the `.eslintrc.json` to the root of the project will fix that issue (from memory we were just trying avoid adding yet another file there).

**Comment by zalog** [comment]
Yes, no other files, they are hard to maintain.
I'll try to make a new PR with this, this days.

**Pull request closed without merging**